tests/test_trnsys.py

```diff
@@ -395,6 +395,68 @@ def test_t_initial_to_b18(self, config, converttesteasy):
         # Asserts initial temperature is written in b18_lines
         assert any("TINITIAL= 18" in mystring for mystring in b18_lines[200:])
 
+    def test_closest_coords(self, config, converttesteasy):
+        output_folder = None
+        # Deletes temp
+        if os.path.exists(settings.cache_folder):
+            shutil.rmtree(settings.cache_folder)
+        (
+            idf,
+            idf_file,
+            weather_file,
+            window_lib,
+            trnsidf_exe,
+            template,
+            kwargs,
+        ) = converttesteasy
+        try:
+            (
+                idf_file,
+                weather_file,
+                window_lib,
+                output_folder,
+                trnsidf_exe,
+                template,
+            ) = _assert_files(
+                idf_file, weather_file, window_lib, output_folder, trnsidf_exe, template
+            )
+        except:
+            output_folder = os.path.relpath(settings.data_folder)
+            print("Could not assert all paths exist - OK for this test")
```

Comment:
What can cause this test to fail?

Reply:
If the assertion do not pass

Reply:
If you know a certain bit of code will fail, it is better to put it inside a `with pytest.raises(Exception):`